ts/features/wallet/onboarding/cobadge/screens/start/CoBadgeChosenBankScreen.tsx

```diff
@@ -46,25 +78,44 @@ const CoBadgeChosenBankScreen = (props: Props): React.ReactElement => {
   const abiInfo: Abi | undefined = props.abiList.find(
     aI => aI.abi === props.abi
   );
-  return (
-    <SearchStartScreen
-      testID={props.testID}
-      methodType={"cobadge"}
-      onCancel={props.onCancel}
-      onSearch={props.searchAccounts}
-      handlePartecipatingBanksModal={openPartecipatingBankModal}
-      handleTosModal={openTosModal}
-      bankName={abiInfo?.name}
-    />
+
+  if (
+    abiInfo === undefined &&
+    props.abi !== undefined &&
+    isReady(props.abiListStatus)
+  ) {
+    props.onGeneralError("Abi not found in abilist in CoBadgeChosenBankScreen");
+    return null;
```

Comment:
```suggestion
    void mixpanelTrack("BANCOMAT_DETAIL_NO_ABI_ERROR", {
      issuerAbiCode: props.abi
    });
    return null;
```
With the current implementation, the tracking of the missing ABI from the ABI list should be moved here.
We have also the `WALLET_ONBOARDING_COBADGE_FAILURE`, but the action `BANCOMAT_DETAIL_NO_ABI_ERROR` could be useful in order to have a list of missing abi (The user have a bancomat with an abi not in list).

Reply:
Added in [372428c](https://github.com/pagopa/io-app/pull/2940/commits/372428cd2aa7596b2763840d8f738695728c82f5)